test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -350,7 +348,6 @@
                    msg-matcher (re-pattern (str "Path " expected-path " exists but.*"))]
                (is (= 500 (:status response)))
                (is (re-matches msg-matcher (:body response)))
-               (is (= "text/plain" (get-in response [:headers "Content-Type"])))
```

Comment:
This seems odd. The 500 should return some sort of error and a Content-Type along with it. It may be that you'll need to add the content type specifically to the 500 error (which has a body).